ros2/src/autorccar_costmap/include/costmap.h

```diff
@@ -0,0 +1,82 @@
+#ifndef COSTMAP_H
+#define COSTMAP_H
+
+#include <pcl/common/transforms.h>
+#include <pcl/filters/passthrough.h>
+#include <pcl/point_cloud.h>
+#include <pcl/point_types.h>
+
+#include <algorithm>
+#include <eigen3/Eigen/Dense>
+#include <fstream>
+#include <iostream>
+#include <string>
+#include <utility>
+#include <vector>
+
+struct CostmapInfo {
+    double resolution;    // map resolution [m/cell]
+    unsigned int size_x;  // map width [cell]
+    unsigned int size_y;  // map height [cell]
+    double origin_pos_x;  // real-world pose of the cell (0,0) in the map.
+    double origin_pos_y;
+    Eigen::MatrixXd *costmap;
+};
+
+class Costmap {
+   public:
+    Costmap(const std::string &config_file_path);
+
+    bool costmap_flag;  // noticify it's time to update the costmap
```

Comment:
```suggestion
    bool costmap_flag_;  // noticify it's time to update the costmap
```

class 멤버 변수에는 trailing underscore 가 붙습니다
https://google.github.io/styleguide/cppguide.html#Variable_Names


vscode 에서  naming 안맞으면 warning을 주는게 있는데 활성화가 필요하겠군요...
vscode setting에서 clang tidy 키면 되는것 같긴 한데 자세한건 잘 모르겠씁니다.

변수명 바꾼다면 하는김에 =false 초기화도...